.travis.yml

```diff
@@ -1,26 +1,42 @@
 language: generic
+dist: trusty
```

Comment:
Oh? That's pretty far behind... Alright, it can stay. Trusty is old enough as it is.